sql/hive-thriftserver/src/main/scala/org/apache/spark/sql/hive/thriftserver/ui/ThriftServerSessionPage.scala

```diff
@@ -100,7 +100,8 @@ private[ui] class ThriftServerSessionPage(parent: ThriftServerTab)
           <td>{info.groupId}</td>
           <td>{formatDate(info.startTimestamp)}</td>
           <td>{formatDate(info.finishTimestamp)}</td>
-          <td>{formatDurationOption(Some(info.totalTime))}</td>
```

Comment:
@srowen This change to backport [SPARK-28599](https://github.com/apache/spark/pull/25892). 
Or we should revert this change to make another PR to make it clear?

Reply:
I see, these are really duplicates? I would first merge https://github.com/apache/spark/pull/25892 then, but, hm, how is it not a merge conflict? I'm missing something.

My larger point was: does this actually mean you sort all of the entries when there are many pages of them? or just the current page?

Reply:
https://github.com/apache/spark/pull/25855 for `ThriftServerPage`, https://github.com/apache/spark/pull/25892 for `ThriftServerSessionPage`. We can support all pages in `ThriftServerTab` after this PR.

Reply:
Oh I see. Well, want to at least merge #25892 first and then include it here? we can back-port it separately, too.

Reply:
I'll review #25892 now and merge it soon. And, +1 for backporting separately.

Reply:
Ok. I have reverted my last commit and created another PR https://github.com/apache/spark/pull/25893 to back-port [SPARK-28588](https://issues.apache.org/jira/browse/SPARK-28599) this to branch-2.4. 
Kindly review it